File: Pemrograman 1/MODUL 2/Bahasa Python/PRAK205-2410817110002-Akhmad_Daffa_Azzikri.py
import math

inputs = input().split()

if len(inputs) == 2:
    a = float(inputs[0])
    b = float(inputs[1])
else:
    a = float(inputs[0])
    b = float(input())
alas = math.sqrt(b**2 - a**2)
tinggi = a
keliling = a + b + alas
luas = 0.5 * alas * tinggi

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Pemrograman 1/MODUL 2/Bahasa C/PRAK205-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>
#include <math.h>

int main() {
    float a, b, alas, tinggi, keliling, luas;
    
    printf("");
    scanf("%f", &a);
    printf("");
    scanf("%f", &b);
    
    alas = sqrt(b * b - a * a);
    tinggi = a;
    keliling = a + b + alas;
    luas = 0.5 * alas * tinggi;
    
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    return 0;
}
